services/catalog/src/simcore_service_catalog/services/access_rights.py

```diff
@@ -106,7 +106,7 @@ async def evaluate_auto_upgrade_policy(
     # AUTO-UPGRADE PATCH policy:
     #
     #  - Any new patch released, inherits the access rights from previous compatible version
-    #  - TODO: add as option in the publication contract, i.e. in ServiceDockerData
+    #  - IDEA: add as option in the publication contract, i.e. in ServiceDockerData?
```

Comment:
mouarf... ;)

Reply:
it is that an expression of love or discust :-D